main.go

```diff
@@ -451,8 +482,26 @@ func (m *manager) getNetworkNames() ([]string, error) {
 	}
 
 	for _, network := range networks {
-		if (network.Attachable || network.Scope == "local") && !containsString(ignoredNetworkNames, network.Name) {
-			names = append(names, network.Name)
+		// Inspect each network to get attached containers
+		attached, err := m.cli.NetworkInspect(m.ctx, network.Name, types.NetworkInspectOptions{})
+		if err != nil {
+			return nil, err
+		}
+
+		// Count currently attached containerNames
+		attachedContainerNames := 0
+
+		for _, container := range containerNames {
+			if containerInNetworks(container, attached) {
+				attachedContainerNames = attachedContainerNames + 1
+			}
+		}
+
+		// If more than 0 containers in a network (excluding containers from config)
+		if (len(attached.Containers) - attachedContainerNames) > 0 {
+			if (network.Attachable || network.Scope == "local") && !containsString(ignoredNetworkNames, network.Name) {
+				names = append(names, network.Name)
+			}
 		}
```

Comment:
```suggestion
		for _, endpoint := range attached.Containers {
			if !containsString(containerNames, endpoint.Name) &&
				(network.Attachable || network.Scope == "local") &&
				!containsString(ignoredNetworkNames, network.Name) {
				names = append(names, network.Name)
				break
			}
		}
```
I think this is simpler.